documentation/modules/ROOT/pages/operations/openshift.adoc

```diff
@@ -8,9 +8,7 @@
 
 toc::[]
 
-The following describes how to set up the {prodname} connectors for change data capture on Red Hat's https://www.openshift.com/[OpenShift] container platform.
-
-These instructions have been tested by using the https://github.com/minishift/minishift[Minishift tool], which allows you to run a single node OpenShift instance locally on your machine.
+This procedure is for setting up {prodname} connectors on Red Hat's https://www.openshift.com/[OpenShift] container platform. These instructions have been tested by using the https://github.com/minishift/minishift[Minishift tool], which allows you to run a single node OpenShift instance locally on your machine.
 
 A complete example of this set-up using Minishift is in the https://github.com/debezium/debezium-examples/tree/master/openshift[{prodname} examples repository].
```

Comment:
The minishift sample in the examples repository still refers to s2i.

Reply:
Minifshift should no longer be referenced since it doesn't work with OCP 4.x. Either rewrite this to "Tested with two latest releases of OpenShift 4.x" or drop the line about testing.

Reply:
I removed the references to minishift. I also removed the sentence that starts with "A complete example..." since that example uses minishift.